imports/plugins/custom/products-review/server/methods/productReview.js

```diff
@@ -0,0 +1,22 @@
+import {
+  Meteor
+} from "meteor/meteor";
+import { Reviews } from "../../lib/collections";
+import { check } from "meteor/check";
+/**
+    * Methods
+    * @param {Object}
+    * @returns {Object} Ratings and reviews
+ */
```

Comment:
Doesn't follow your set convention. Be consistent.

Reply:
Thanks for the review @bekomay26